tests/schemas/valid/test_string_transforms.json

```diff
@@ -590,35 +606,33 @@
               "question": {
                 "id": "question7",
                 "title": "Title",
-                "description": ["Checkbox option label tests"],
+                "description": ["Format list tests"],
                 "type": "General",
                 "answers": [
                   {
                     "id": "q7-answer-1",
                     "mandatory": false,
-                    "type": "Checkbox",
-                    "label": "Placeholders in checkbox option label",
-                    "options": [
-                      {
-                        "label": {
-                          "text": "test {simple_answer}",
-                          "placeholders": [
+                    "type": "TextField",
+                    "label": "Simple format list",
+                    "description": {
+                      "text": "{list}",
+                      "placeholders": [
+                        {
+                          "placeholder": "list",
+                          "transforms": [
                             {
-                              "placeholder": "simple_answer",
-                              "value": {
-                                "source": "answers",
-                                "identifier": "ref-answer1"
+                              "transform": "format_list",
+                              "arguments": {
+                                "list_to_format": {
+                                  "source": "answers",
+                                  "identifier": "q5-answer-1"
```

Comment:
This looks like I have changed a lot, all I did was flip block 5 and 7 and update the identifier from a list of answers to a checkbox id. I checked with Donna and v2 to make sure this wasn't used to format different answer to look like a list.